files/fr/web/css/@font-face/descent-override/index.html

```diff
@@ -0,0 +1,74 @@
+---
+title: descent-override
+slug: Web/CSS/@font-face/descent-override
+browser-compat: css.at-rules.font-face.descent-override
+translation_of: 'Web/CSS/@font-face/descent-override'
+---
+<p>{{CSSRef}}</p>
+
+<p>Le descripteur CSS <strong><code>descent-override</code></strong> définit la mesure du jambage inférieur (descendant) de la police. La mesure du jambage inférieur correspond à la distance verticale par rapport à la hauteur d'X (hauteur du caractère bas de casse, à l'exclusion des jambages inférieurs et supérieurs) que CSS doit utiliser pour agencer les boîtes dans un contexte de formatage en ligne.</p>
+
+<h2 id="syntax">Syntaxe</h2>
+
+<pre class="brush: css">descent-override: normal;
+descent-override: 90%;</pre>
+
+<h3 id="values">Valeurs</h3>
+
+<dl>
+  <dt><code>normal</code></dt>
+  <dd>La valeur par défaut. La mesure du jambage inférieur est obtenue à partir du fichier de police.</dd>
+  <dt><code>&lt;percentage&gt;</code></dt>
+  <dd>Une valeur exprimée en {{cssxref("&lt;percentage&gt;", "&lt;pourcentage&gt;")}}.</dd>
```

Comment:
Same as https://github.com/mdn/translated-content/pull/1752/files#r678016933 :)

Reply:
Fixed in https://github.com/mdn/translated-content/pull/1753/commits/aea8cad8466742d4670b37aa38a80c1ac2149371